etl/steps/data/grapher/cait/2022-08-10/carbon_dioxide_emissions_by_sector.py

```diff
@@ -0,0 +1,38 @@
+from owid import catalog
+
```

Comment:
Isn't this dataset identical to `cait/2022-08-10/carbon_dioxide_emissions_by_sector`? I guess your plan was to remove `co2_emissions`, but you forgot?

Reply:
Or, alternatively, for consistency with `ch4_emissions` and `n2o_emissions`, why not removing `carbon_dioxide_emissions_by_sector`? Maybe I'm misunderstanding the problem.

Reply:
My bad, I originally thought I'd rename those files to keep `short_name` same as in our DB, but then I changed my mind and forgot to delete it.